src/projects/providers/webrtc/webrtc_provider.h

```diff
@@ -123,5 +127,9 @@ namespace pvd
 		std::shared_ptr<Certificate> _certificate = nullptr;
 
 		std::mutex _stream_lock;
+
+		mutable std::shared_mutex _streams_guard;
+		// Key: stream_key / Value: WebRTCStream
+		std::map<ov::String, std::shared_ptr<WebRTCStream>> _streams;
```

Comment:
The name _streams is also not very clear.
This is a table to quickly find the stream mapped with the whip session key, so change it to such a variable name.